### PR TITLE
feature: adding aggregations

### PR DESCRIPTION
**Pull request by lorenzo-solcast**
Adding a module `solcast.aggregations` to query the API for live/forecast aggregations. It has no test so far as we are yet to expose _unmetered_ aggregations.

### CODE REVIEW

**Review by michaelf-solcast — COMMENTED** [review]
Is there an endpoint that lists the collections and aggregations available to the user?  It can be a challenge with aggs to work out what collection_id and aggregation_id to use.

**Comment by lorenzo-solcast** [comment]
there is but is not a public API (yet). Suggest we don't do not add until it is exposed in the API docs.

File: solcast/__init__.py [hunk 1]
```diff
@@ -1,17 +1,19 @@
-__version__ = "1.2.5"
+__version__ = "1.2.6"
```
Comment (michaelf-solcast) [inline solcast/__init__.py]: For adding a whole new endpoint, should this be a minor version increment i.e. 1.3.0?

File: solcast/aggregations.py [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+    forecast_grid_aggregations,
+    live_grid_aggregations,
+)
+
+
+def live(
+    collection_id: str, aggregation_id: Optional[str], **kwargs
+) -> PandafiableResponse:
+    """
+    Get live aggregation data for up to 7 days of data at a time for a requested collection or aggregation
```
Comment (michaelf-solcast) [inline solcast/aggregations.py]: Add full stop

**Review by michaelf-solcast — COMMENTED** [review]
Also need to add to the list of available modules in index.md.

**Review by michaelf-solcast — APPROVED** [review]

**Pull request merged**